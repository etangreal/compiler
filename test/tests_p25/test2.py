a = 1
b = 2
c = 3
d = 4
e = 5

if ( True and True or False ):
	b = 2

	if ( False ):
		d = 4

	elif( True ):
		d = 44

	else:
		d = 444

	c = 3

elif ( False ):
	b = 22

else:
	b = 222

e = 5

print a
print b
print c
print d
print e