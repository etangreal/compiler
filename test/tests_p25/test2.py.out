1
2
3
44
5
